examples/tools/autotools/autotools_toolchain/build_project_autotools_toolchain.rst

```diff
@@ -0,0 +1,215 @@
+.. _examples_tools_autotools_autotools_toolchain_build_project_autotools_toolchain:
+
+Build a simple Autotools project using Conan
+============================================
+
+In this example, we are going to create a string formatter application
+that uses one of the most popular C++ libraries: `fmt <https://fmt.dev/latest/index.html/>`__.
+
+We'll use Autotools as build system and pkg-config as helper tool in this case, so you should get them installed
+on Linux and Mac before going forward with this example. In case you do not have them installed already, you can
+`Conan Center <https://conan.io/center/>`_ and install them as build requirements.
+
+Please, at first, clone the sources to recreate this project, you can find them in the
+`examples2.0 repository <https://github.com/conan-io/examples2>`_ in GitHub:
+
+.. code-block:: shell
+
+    git clone https://github.com/conan-io/examples2.git
+    cd examples2/examples/tools/autotools/autotoolstoolchain/string_formatter
+
+We start from a very simple C++ language project with this structure:
+
+.. code-block:: text
+
+    .
+    ├── configure.ac
+    ├── Makefile.am
+    ├── conanfile.txt
+    └── src
+        └── main.cpp
+
+This project contains a basic *configure.ac* including the **fmt** pkg-config dependency and the
+source code for the string formatter program in *main.cpp*.
+
+Let's have a look at the *main.cpp* file, it only prints a default message, but uses ``fmt::print`` method for it.
+
+.. code-block:: cpp
+    :caption: **main.c**
+
+    #include <cstdlib>
+    #include <fmt/core.h>
+
+    int main(int argc, char** argv) {
+        fmt::print("{} - The C++ Package Manager!\n", "Conan");
+        return EXIT_SUCCESS;
+    }
+
+The ``configure.ac`` tells us about our project will a C++ compiler and a pkg-config module named as ``fmt.pc``
```

Comment:
This sentence is a bit confusing, can we rephrase this? Something like this?

The ``configure.ac`` file checks for a C++ compiler using the ``AC_PROG_CXX`` macro and also checks for the `fmt.pc` pkg-config module using the PKG_CHECK_MODULES macro.

Reply:
yes, please change. it doesn't `tell`, as it's not living entity, I think it's confusing for non-native speakers to use metaphors.

Reply:
Changed.